### ISSUE TITLE
Problem with "padding-left: 90px;" of footer (copyright) when show big navbar.

### ISSUE THREAD

**Issue by thearabbit** [tui2tone/flat-admin-bootstrap-templates#7]
Problem with "padding-left: 90px;" of footer (copyright) when show big navbar.
```js
--------------
|   Content |
--------------
............ copyright... // It is OK (25px);
```

**Comment by thearabbit** [comment]
Now I fix by
```js
// style.css
.app-container.expanded .app-footer .wrapper { // default don't have .wrapper
    padding-left: 25px;
}

@media (max-width: 768px) {
    .app-container.expanded .app-footer .wrapper { // default don't have .expanded 
        padding-left: 25px;
    }
}
``s

**Comment by tui2tone** [comment]
Fix in v2.1.1
Thank you so much.
